pkg/stores/sqlproxy/proxy_store.go

```diff
@@ -73,8 +74,50 @@ var (
 		gvkKey("", "v1", "Pod"): {
 			{"spec", "containers", "image"},
 			{"spec", "nodeName"}},
+		gvkKey("", "v1", "Service"): {
+			{"spec", "clusterIP"},
+			{"spec", "type"},
+		},
+		gvkKey("networking.k8s.io", "v1", "Ingress"): {
```

Comment:
I would prefer to maintain this list in alpha order, and would have the `"networking.k8s.io"` entry placed after the last `"management.cattle.io"` entry.

Reply:
Done as d78c242